environmental-footprint/environmental-footprint/additional-content/environmental-footprint-calculator.mdx

```diff
@@ -19,45 +19,45 @@ The environmental footprint estimation tool is currently in private beta. Sign u
 
 This page details the methodology used to calculate the environmental footprint of Scaleway cloud services, and Bare Metal offers. This methodology ensures transparency and precision, as it provides reliable estimations covering a full lifecycle analysis.
 
-## Methodology Overview
+## Methodology overview
 
 Our calculation methodology was developed in collaboration with the green IT specialist consultancy firm [IJO](https://ijo.tech/index.html), ensuring a rigorous and expert-validated approach to assessing environmental impact accurately and transparently.
 
-Our Methodology is based on the [ADEME (French Government Agency for Ecological Transition) Product Category Rules (PCR) for Datacenter and Cloud services](https://librairie.ademe.fr/produire-autrement/6105-methodological-standard-for-the-environmental-assessment-of-datacenter-it-hosting-services-and-cloud-services.html), which provide a standardized framework for calculating and reporting the environmental impact of datacenters and cloud services. These rules are designed to ensure consistency, transparency, and comparability in the assessment of environmental footprints across the industry.
+Our Methodology is based on the [ADEME (French Government Agency for Ecological Transition) Product Category Rules (PCR) for Data center and Cloud services](https://librairie.ademe.fr/produire-autrement/6105-methodological-standard-for-the-environmental-assessment-of-datacenter-it-hosting-services-and-cloud-services.html), which provide a standardized framework for calculating and reporting the environmental impact of data centers and cloud services. These rules are designed to ensure consistency, transparency, and comparability in the assessment of environmental footprints across the industry.
```

Comment:
Same here, when referring to the title of the document maybe we should leave it the way they write it (Datacenter)